README.md

```diff
@@ -1 +1,53 @@
-Observe Kubernetes events, collect metrics & trigger endpoint receivers
\ No newline at end of file
+# ![logo](kubeobserver_logo.png)
+
+Observe Kubernetes events, collect metrics & trigger endpoint receivers
+
+## Quick Start
+
+#### Build from Scratch
+
+```bash
+$ make build
+$ ./kubeobserver
+```
+
+## Configuration
+
+### Kubeobserver configuration
+
+Kubeobserver is configure throw environment variables.
+
+| Variable name | Mandatory | Description | Default |
+| --- | --- | --- | --- |
+| K8S_CLUSTER_NAME | true | the cluster name kubeobserver deployed to (for example: "dev-clsuster") | - |
```

Comment:
consider "the name of the k8s cluster Kubeobserver is deployed to (for example: "dev-cluster")"? mind typo in "dev-cluster"